index.html

```diff
@@ -20,12 +20,6 @@
 <body onload="run();">
   <h1>unitncal</h1>
   <p><strong>Sincronizza nativamente il tuo calendario con gli orari dell'università di Trento!</strong></p>
```

Comment:
Non sarebbe meglio scrivere "Sincronizza costantemente", rende più chiaro il senso del programma?

Reply:
Beh, "costantemente" dovrebbe essere già implicato da "sincronizza". Il punto di forza di questo tool è che il calendario è in un formato standard e quindi si può usare il supporto nativo del proprio calendario, senza applicazioni aggiuntive